src/style.css

```diff
@@ -1,10 +1,14 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
 body {
-  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
-  font-size: 16px;
-}
+  font-family: 'Roboto', sans-serif;
+  font-size: 13px;
+  background-image: url('/assets/patrick-tomasso-QMDap1TAu0g-unsplash.jpg');
+  background-repeat: no-repeat;
+  object-fit: cover;
 
-h1 {
-  margin: 3rem;
+  /* background-color: #151b29;
+  background-color:#2a0e2b; */
```

Comment:
- [ ] **[OPTIONAL]** Why not remove all the unnecessary commented lines of code to make your codebase cleaner and more professional?

Reply:
Thanks I always forget that.:)